conf.py

```diff
@@ -43,6 +43,19 @@
         "Please make sure you read https://exercism.org/docs/community/"
         "being-a-good-community-member/suggesting-exercise-improvements before posting."
     ),
+    "beginner": (
+        "👋🏼  Hello! We are happy to have you here." 
+        "Exercism is best suited for people who have some experience in programming. "
+        "If you are completely new to the topic, exercism might be very challenging. "
+        "You should definitely try the mentoring function to get the most out of exercism. "
```

Comment:
```suggestion
```

Reply:
So, remove the mention of mentoring, or get better wording?